src/js/components/Clock/doc.js

```diff
@@ -52,3 +52,76 @@ export const doc = Clock => {
 
   return DocumentedClock;
 };
+
+export const themeDoc = {
+  'clock.analog.extend': {
+    description: 'Any additional style for Clock.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'clock.analog.hour.color': {
+    description: 'The color of the hour hand.',
+    type: 'string',
```

Comment:
let's make sure `color` prop has the dark/light object as well on docs
`type: 'string | { dark: string, light: string }',`